cfp/index.html

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+---
+<article id="home">
+  <dl>
+    {% assign now = site.time | date: '%s' | plus: 0 %}
+    {% assign conferences = site.data.conferences | sort: 'start_date' %}
+    {% for event in conferences %}
+      {% assign start_date = event.start_date | date: '%s' | plus: 0 %}
+      {% assign cfp_date = event.cfp_date | date: '%s' | plus: 0 %}
+      {% if start_date > now and cfp_date > now %}
```

Comment:
I'm wondering if this should be `>=` for both of them?

Right now, if the CFP closes today this wouldn't be shown, right?

Reply:
Agreed? I think? Just stole the logic from the "home" view.

How can I be expected to do date logic without tests? Impossible.

Reply:
```suggestion
      {% if start_date >= now and cfp_date >= now %}
```

The `start_date` condition is probably redundant too.

Reply:
Actually the `>=` also doesn't help in this case. `now` is a UNIX timestamp with seconds (like `Time.current.to_i`), and `cfp_date` is treated as a UNIX timestamp without seconds like  (`Date.parse("2024-01-24").to_time.to_i`).

So, even for a CFP that ends today, it won't be shown since the `cfp_date` at midnight is smaller than `now`. Wondering if there's something like `today` in Jekyll which is also just a `Date`.

Reply:
But then again, maybe it would just make sense to also include the Time information in the `cfp_date`